From 0720c7bef71208a5fe10ffceedc6888655a5d75c Mon Sep 17 00:00:00 2001
From: Theo GOUREAU <theo.goureau-ext@st.com>
Date: Wed, 8 Jan 2025 14:18:28 +0100
Subject: [PATCH 1438/1695] dts: stm32: change LPTIM4 clock for LSI for
 stm32mp257f-ev1

Change LPTIM4 clock for LSI for stm32mp257f-ev1
needed for STm32Cube FW M0+ demo in standby

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c0c0c782c4b8d6fc2eb24d3bc417695ccf20f964 dts: stm32: change LPTIM4 clock for LSI for stm32mp257f-ev1]
Change-Id: I17932fba0ccbdc0934c431e6e4a8dd07a9ef09ba
Signed-off-by: Theo GOUREAU <theo.goureau-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/427549
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index c246ad101..8e64ea319 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -75,7 +75,7 @@
 		FLEXGEN_CFG(38, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(39, XBAR_SRC_MSI_KER, 0, 0)
 		FLEXGEN_CFG(40, XBAR_SRC_LSE, 0, 0)
-		FLEXGEN_CFG(41, XBAR_SRC_PLL4, 0, 11)
+		FLEXGEN_CFG(41, XBAR_SRC_LSI, 0, 0)
 		FLEXGEN_CFG(42, XBAR_SRC_PLL7, 0, 16)
 		FLEXGEN_CFG(43, XBAR_SRC_PLL4, 0, 23)
 		FLEXGEN_CFG(44, XBAR_SRC_PLL4, 0, 5)
-- 
2.47.2

